externalpackages/PVODE/include/pvode/nvector.h

```diff
@@ -88,12 +88,23 @@ namespace pvode {
 
 
 /* Set types real and integer for MPI calls. */
-
+#ifndef PVEC_REAL_MPI_TYPE
 #if (LLNL_DOUBLE == 1)
 #define PVEC_REAL_MPI_TYPE MPI_DOUBLE
 #else
 #define PVEC_REAL_MPI_TYPE MPI_FLOAT
 #endif
+#else
+#if (LLNL_DOUBLE == 1)
+#if PVEC_REAL_MPI_TYPE != MPI_DOUBLE
```

Comment:
This is causing problems in CI, can we just drop the `#else` branch here? The `PVCE_INTEGER_MPI_TYPE` bit below doesn't have it and that seems to not cause problems

Reply:
Sure - it was working for me, so I thought lets keep it there to avoid a mismatch ...